frontend/src/locales/en.yml

```diff
@@ -523,3 +541,111 @@ anomaly:
+      title: Version control workflow
+      content: >-
+        Database migration scripts are stored in a git repository. To
+        make schema changes, a developer would create a migration script
+        and submit for review in the corresponding VCS such as GitLab.
+        After the script is approved and merged into the configured
+        branch, Bytebase will automatically kicks off the task to apply
+        the new schema change.
+    configure-version-control: Configure version control
+    wizard:
+      enabled-version-control: Successfully enabled version control workflow for {projectName}
+      changed-repo: Successfully changed repository for {projectName}
+  overview:
+    view-all: View all
+    view-all-closed: View all closed
+    recent-activity: "Recent @.lower:{'common.activities'}"
+    in-progress: In progress
+    recently-closed: Recently Closed
+    info-slot-content: >-
+      Bytebase periodically syncs the instance schema.
+      Newly synced databases are first placed in this default project.
+      User should transfer them to the proper application project.
+    no-db-prompt: >-
+      No database belongs to this project.
+      You can add database by clicking {newDb} or {transferInDb} from the top action bar.
+  webhook:
+    create-webhook: Add a webhook
+    last-updated-by: Last updated bu
+    destination: Destination
+    webhook-url: Webhook url
+    triggering-activity: Triggering activities
+    test-webhook: Test Webhook
+    no-webhook:
+      title: No webhook configured for this project.
+      content: >-
+        Configure webhooks to let Bytebase post notification to the external
+        systems on various events.
+    creation:
+      title: Create webhook
+      desc:  >-
+        Create the corresponding webhook for the {destination} channel receiving the message.
+      how-to-protect: >-
+        If you want to use keyword list to protect the webhook, you can add
+        "Bytebase" to that list.
+      view-doc: View {destination}'s doc
+    deletion:
+      btn-text: Delete this webhook
+    activity-item:
+      issue-creation:
+        title: Issue creation
+        label: When new issue has been created
+      issue-status-change:
+        title: Issue status change
+        label: When issue status has changed
+      issue-task-status-change:
+        title: Issue task status change
+        label: When issue's enclosing task status has changed
+      issue-info-change:
+        title: Issue info change
+        label: When issue info (e.g. assignee, title, description) has changed
+      issue-comment-creation:
+        title: Issue comment creation
+        label: When new issue comment has been created
+  settings:
+    member-placeholder: Select user
+    manage-member: Manage members
+    add-member: Add member
+    owner: "@.upper:common.role.owner"
+    developer: "@.upper:common.role.developer"
+    archive:
+      btn-text: Archive this project
```

Comment:
nit: can use lower modifier and reference @:common.project https://kazupon.github.io/vue-i18n/guide/messages.html#formatting-linked-locale-messages

Same below